### PR TITLE
Update prerequisites

### PR DESCRIPTION
**Pull request by mervey45**
-

### CODE REVIEW

File: tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md [hunk 1]
```diff
@@ -276,7 +275,9 @@ author_profile: https://github.com/mervey45
 
       Click **OK**.
 
-      **Save** and **activate** your IAM app.
+      Right-click your IAM App and click **Activate**.
+
+      **HINT:** If you can't find your service name, you can press **CTRL + space** to search for it.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:280]: Please be consistent with upper/lower case within your tutorials.

File: tutorials/abap-s4hanacloud-procurement-purchasereq-shop/abap-s4hanacloud-procurement-purchasereq-shop.md [hunk 2]
```diff
@@ -120,479 +119,471 @@ In this tutorial, wherever XXX appears, use a number (e.g. 000).
...
-  10. Click **Finish**.
+  3. Maintain the required information on the **Configure Generator** dialog to provide the name of your data model and generate them.         
+     
+     For that, navigate through the wizard tree **(Business Objects, Data Model, etc...)**, maintain the artefact names provided in the table below, and press **Next >**.
 
-       ![table](table6.png)
+     Verify the maintained entries and press **Next >** to confirm. The needed artifacts will be generated.
 
-  11. Replace your code with following:
+     **Please note**: Error Invalid XML format.
```
Comment (MichaelCzcz) [inline tutorials/abap-s4hanacloud-procurement-purchasereq-shop/abap-s4hanacloud-procurement-purchasereq-shop.md:175]: Please use formatting for notes.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by mervey45** [comment]
@MichaelCzcz I updated the hint sections

**Comment by MichaelCzcz** [comment]
@mervey45 Please resolve conflicts

**Comment by jmmargo** [comment]
Merve, If I understand correctly, Michael would prefer you use the ">" syntax for notes instead of saying "Please note."  Anyplace where it is important to note something, the best practice is to use ">" followed by the text.  You do not need words like "Note" or "Please note" or "Hint" as the text will be visually offset with the ">" function.

**Comment by mervey45** [comment]
@jmmargo I changed the hint to >

**Comment by jmmargo** [comment]
Merve, thank you for adding the note syntax; however, as there is a space between the ">" and the text that follows, it is not rendering correctly.  You can double check this in QA Blue.  There should not be a space between the ">" and the following text.

**Comment by mervey45** [comment]
@jmmargo I deleted the space between the text and >

**Comment by jmmargo** [comment]
Thank you for rmaking the changes.

**Pull request merged**